src/utils.ts

```diff
@@ -68,6 +66,10 @@ export class Utils {
         }
     }
 
+    public static async removeArtifactoryServers(cliPath: string) {
+        await Utils.runCli(cliPath, ['c', 'rm', '--quiet']);
```

Comment:
TODO - check the CLI version and run `jfrog rt c clean` or `jfrog rt c rm` accordingly.